mysql-test/suite/innodb/r/page_cleaner.result

```diff
@@ -3,12 +3,20 @@ SET @save_pct_lwm= @@GLOBAL.innodb_max_dirty_pages_pct_lwm;
 SET GLOBAL innodb_max_dirty_pages_pct_lwm=0.0;
 SET GLOBAL innodb_max_dirty_pages_pct=0.0;
 SET GLOBAL innodb_max_dirty_pages_pct=90.0;
+select variable_value into @innodb_async_writes_total_count_before
+from information_schema.global_status
+where variable_name='innodb_async_writes_total_count';
 CREATE TABLE t ENGINE=InnoDB SELECT * FROM seq_1_to_10000;
 SELECT variable_value>0 FROM information_schema.global_status
 WHERE variable_name = 'INNODB_BUFFER_POOL_PAGES_DIRTY';
 variable_value>0
 1
 SET GLOBAL innodb_max_dirty_pages_pct=0.0;
+select (variable_value > @innodb_async_writes_total_count_before) as innodb_async_writes_grow
```

Comment:
This will not work reliably: `'10'>'9'` does not hold. Yes, for some reason, `variable_value` is of a string type. Maybe `variable_value - @before > 0` would work, or you should use explicit conversions for both comparands, like this:
```sql
SELECT CAST(VARIABLE_VALUE AS INTEGER) INTO @var /* … */;
```
It took me quite some effort to figure out the causes of some sporadic test failures due to this.

Reply:
Thanks for figuring it out. Fixed now.